frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/test-expand.scenario.ts

```diff
@@ -3,7 +3,7 @@ import { clickKebabAction } from '@console/internal-integration-tests/views/crud
 import { click } from '@console/shared/src/test-utils/utils';
 import { testName } from '@console/internal-integration-tests/protractor.conf';
 import { testPVC, testDeployment } from '../../mocks/expand-test-mocks';
-import { requestSizeValueInput } from '../../views/add-capacity.view';
+import { capacityValueInput } from '../../views/add-capacity.view';
```

Comment:
You need to import `currentSelectors` and from that, you get the selector you need. Refer to install-flow tests.

Reply:
As mentioned in the description, selectors specific changes I am planning to send as part of follow up PR.

Reply:
Okay so what is the point of this PR?

Reply:
As it requires selectors across multiple version of OCP and needs to be tested for each

Reply:
The intention of this PR is to add tests for add capacity for baremetal infra. I will update the description. Thanks!